R/connect.R

```diff
@@ -132,6 +132,9 @@ Connect <- R6::R6Class(
     #' `httr::content(res, as = parser)`.
     #' @param ... Additional arguments passed to the request function
     request = function(method, url, ..., parser = "parsed") {
+      old_opt <- options(scipen = 999)
+      on.exit(options(old_opt), add = TRUE)
```

Comment:
💯 Even got the various slightly funky things around `options()` (that it returns its former self!) and `add` with `on.exit()` (why that is not default I will never know!)